english/TodayWasAGoodDay

```diff
@@ -0,0 +1 @@
+Today was a good day! Keep your head up, dream big and work hard to achieve it. Bless up!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.